tubesize/src/components/options.tsx

```
@@ -0,0 +1,132 @@
+                        }}
+                    >
+                        <option value="1">1 Day</option>
+                        <option value="3">3 Days</option>
+                        <option value="7">7 Days</option>
+                    </select>
+                </div>
+                <button
+                    id="resetCache"
+                    className="reset-cache-btn"
+                    onChange={async () => await chrome.storage.local.clear()}
+                >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)